provider/cmd/pulumi-resource-nuage/nuage_provider/provider.py

```diff
@@ -107,19 +96,22 @@ def _create_repository(
     )
 
 
-def _create_container(
-    name: str, inputs: Inputs, options: Optional[ResourceOptions] = None
-) -> ConstructResult:
-    created_container = ContainerFunction(
-        name, ContainerFunctionArgs.from_inputs(inputs), dict(inputs), options
+def _create_image(name: str, inputs: Inputs, options: Optional[ResourceOptions] = None) -> ConstructResult:
+    created_image = Image(name, ImageArgs.from_inputs(inputs), dict(inputs), options)
+    return provider.ConstructResult(
+        urn=created_image.urn,
+        state={"name": created_image.name, "uri": created_image.uri},
     )
 
+
+def _create_container(name: str, inputs: Inputs, options: Optional[ResourceOptions] = None) -> ConstructResult:
```

Comment:
Maybe rename to create function ?

Container is confusing now that we have an Image